lua/orgmode/objects/date.lua

```diff
@@ -858,8 +898,9 @@ function Date:get_repeater()
   return nil
 end
 
+---@return OrgDate
 function Date:set_todays_date()
-  local time = os.date('*t', os.time())
+  local time = os_date(os.time())
```

Comment:
Same as other comment, should we use `os_time()` here?

Reply:
Done.